ports/arkovstower/port.json

```diff
@@ -12,7 +12,7 @@
       "Troidem"
     ],
     "desc": "This roguelike puts you in the shoes of an unnamed adventurer who just woke up on the top level of Arkov's Tower. Try to fight your way down and escape it - or die trying. Move your player trough numerous procedural generated levels. Fight numerous enemies including some unique bosses. The turn based combat system allows you to take your time and plan every move carefully. Upgrade your character with all sorts of items found in chests or dropped by enemies. This game is highly customizable for people with pixel art and/or programming experience.",
-    "inst": "Download the game https://store.steampowered.com/app/2688600/Descent_from_Arkovs_tower_Prologue/ and copy the exe file into game folder.",
+    "inst": "Download [Descent from Arkov's tower Prologue](https://store.steampowered.com/app/2688600/Descent_from_Arkovs_tower_Prologue/) from Steam and copy the exe file into game folder.",
```

Comment:
These urls wont work properly in the PortMaster-GUI. :(

Reply:
and the way it is, it shows up like this on the website :/ 
![image](https://github.com/PortsMaster/PortMaster-New/assets/28956861/63ac7852-e4f3-4564-8969-5941c0add530)
So, I'll take it out and we need to look at the markdown to html for the website?

Reply:
> These urls wont work properly in the PortMaster-GUI. :(

We could add mkd_inst and mkd_desc and have the website use them if they are set.